Lib/fontTools/misc/transform.py

```diff
@@ -450,21 +462,20 @@ def fromTransform(self, transform):
 
         rotation = 0
         scaleX = scaleY = 0
-        skewX = skewY = 0
+        skewX = 0
 
         # Apply the QR-like decomposition.
         if a != 0 or b != 0:
             r = math.sqrt(a * a + b * b)
             rotation = math.acos(a / r) if b >= 0 else -math.acos(a / r)
             scaleX, scaleY = (r, delta / r)
-            skewX, skewY = (math.atan((a * c + b * d) / (r * r)), 0)
+            skewX = math.atan((a * c + b * d) / (r * r))
         elif c != 0 or d != 0:
             s = math.sqrt(c * c + d * d)
             rotation = math.pi / 2 - (
                 math.acos(-c / s) if d >= 0 else -math.acos(c / s)
             )
             scaleX, scaleY = (delta / s, s)
-            skewX, skewY = (0, math.atan((a * c + b * d) / (s * s)))
```

Comment:
Are you saying it's never possible for this math.atan() expression to return anything but zero?

Reply:
Yes. No?

Reply:
Ah, it is because we know that both a and b are zero at this point, right?

Reply:
Exactly.